client/src/app/footer/footer.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="footer-container" fxLayoutAlign="center center">
+  <div>
+    <p class="text">K-App <a href="https://github.com/K-Fet/K-App/releases">v.1.3.0</a> | <strong>Serviam Indeclinabilem</strong> | © 2018<br />
```

Comment:
Let's make the version dynamic (through environnement like recaptacha) with a version in dev (`v1.3.0-dev`) and one in prod `v1.3.1`

Reply:
Why dev version < prod version ?
Btw, good idea about environnement !

Reply:
Ah ok, nevermind !